src/promo/DonationBanner.jsx

```diff
@@ -5,15 +5,16 @@ import Paper from '@material-ui/core/Paper'
 import PropTypes from 'prop-types'
 import React from 'react'
 import Typography from '../Typography'
+import corePalette from '../styles/palettes/core'
 import withStyles from '@material-ui/core/styles/withStyles'
 
 const styles = theme => ({
   // TODO: this is temporary while we work out what to do about
   // themes and palettes properly
   paper: {
     position: 'relative',
-    backgroundColor: theme.palette.core && theme.palette.core.main,
-    color: theme.palette.primary.contrastText,
+    backgroundColor: corePalette[600],
+    color: '#fff',
```

Comment:
Could this be color: neutralPalette[0] ?

Reply:
Yes. It defs can. 👏